### ISSUE TITLE
ValueError: 'roTating' is not a valid RecordingType

### ISSUE THREAD

**Issue by redtailtech** [briis/unifiprotect#428]
### Home Assistant Version?

2021.12.2

### UniFi Protect Version?

1.20.1

### UniFi Protect HA Integration version?

0.11.3

### What happened?

After updating to the latest integration version I'm now getting this error

ValueError: 'roTating' is not a valid RecordingType

### Relevant log output

```shell
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 313, in async_setup
    result = await component.async_setup_entry(hass, self)  # type: ignore
  File "/config/custom_components/unifiprotect/__init__.py", line 189, in async_setup_entry
    nvr_info = await protect.get_nvr()
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/api.py", line 814, in get_nvr
    return NVR.from_unifi_dict(**data, api=self)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 100, in from_unifi_dict
    data = cls.unifi_dict_to_dict(data)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/nvr.py", line 642, in unifi_dict_to_dict
    return super().unifi_dict_to_dict(data)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 595, in unifi_dict_to_dict
    return super().unifi_dict_to_dict(data)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 322, in unifi_dict_to_dict
    data[key] = cls._clean_protect_obj(data[key], unifi_objs[key], api)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 261, in _clean_protect_obj
    return klass.unifi_dict_to_dict(data=data)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/nvr.py", line 568, in unifi_dict_to_dict
    return super().unifi_dict_to_dict(data)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 322, in unifi_dict_to_dict
    data[key] = cls._clean_protect_obj(data[key], unifi_objs[key], api)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 261, in _clean_protect_obj
    return klass.unifi_dict_to_dict(data=data)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 327, in unifi_dict_to_dict
    data[key] = cls._clean_protect_obj_list(data[key], unifi_lists[key], api)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 269, in _clean_protect_obj_list
    items[index] = cls._clean_protect_obj(item, klass, api)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 261, in _clean_protect_obj
    return klass.unifi_dict_to_dict(data=data)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 315, in unifi_dict_to_dict
    data[key] = convert_unifi_data(data[key], cls.__fields__[key])
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/utils.py", line 177, in convert_unifi_data
    value = field.type_(value)
  File "/usr/local/lib/python3.9/enum.py", line 384, in __call__
    return cls.__new__(cls, value)
  File "/usr/local/lib/python3.9/enum.py", line 702, in __new__
    raise ve_exc
ValueError: 'roTating' is not a valid RecordingType
```

**Comment by AngellusMortis** [comment]
Should be fixed in 0.11.4. Can you please confirm?

**Comment by redtailtech** [comment]
Confirmed working again!! Thank you!!
